test/models/subnet/ipv6_test.rb

```diff
@@ -85,6 +87,12 @@ class Subnet::Ipv6Test < ActiveSupport::TestCase
     assert_includes subnet.known_ips, '2001:db8::4'
   end
 
+  test "should not be able to save with MTU set to 4294967295" do
+    subnet = FactoryBot.build(:subnet_ipv6)
```

Comment:
FactoryBot.build(:subnet_ipv6, :mtu => 4294967295)

Reply:
`FactoryBot.create` if you want to test database.